Vokoder.podspec

```diff
@@ -32,8 +32,6 @@ Pod::Spec.new do |s|
 
   s.subspec 'DataSources' do |ss|
     ss.dependency 'Vokoder/Core'
-    ss.ios.deployment_target = '8.0'
-    ss.tvos.deployment_target = '9.0'
```

Comment:
Would it be better to add watchOS explicitly here and below instead of removing everything?